ui/dashboards/src/components/VariableAutocomplete.tsx

```diff
@@ -25,14 +25,17 @@ export interface VariableAutocompleteProps {
   onChange: (value: string | string[]) => void;
   onOptionsChange: (options: string[]) => void;
   TextFieldProps?: TextFieldProps;
+
+  className?: AutocompleteProps['className'];
+  sx?: AutocompleteProps['sx'];
 }
 
 /**
  * A MUI Autocomplete that displays variable options, loaded from the
  * appropriate plugin.
  */
 export function VariableAutocomplete(props: VariableAutocompleteProps) {
```

Comment:
Curious if you know what's going on with this loading state in nodeExporterDashboard and benchmarkDashboard, is that Interval variable defined correctly?

![loading_state_ex](https://user-images.githubusercontent.com/9369625/183737661-2eb51edd-33d0-4c2b-8433-f1d251dabd50.png)